src/Common/src/Interop/Windows/mincore/Interop.MoveFileEx.cs

```diff
@@ -13,6 +14,8 @@ internal partial class mincore
 
         internal static bool MoveFile(string src, string dst)
         {
+            src = PathInternal.AddExtendedPathPrefixForLongPaths(src);
+            dst = PathInternal.AddExtendedPathPrefixForLongPaths(dst);
```

Comment:
Nit: Why is this P/Invoke called SomethingEx and the others are SomethingPrivate?